src/app/components/TabBar/styles.ts

```diff
@@ -0,0 +1,24 @@
+import { gray, green, white } from '@constants/colors';
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  image: {
+    height: 28,
+    width: 30,
+    marginTop: 3
+  },
+  textActive: {
+    color: green
+  },
+  textInactive: {
+    color: gray
+  },
+  selectedRoute: {
+    padding: 14,
+    backgroundColor: white,
+    marginBottom: 10,
+    borderRadius: 100
```

Comment:
100 is a lot, sure or you need that much?

Reply:
I think it like circle in center tab, if you believe this is unnecessary, i can removed

Reply:
no this is fine, I resolve the comment